fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/CreateUpdateRDFSourcePersisterTest.java

```diff
@@ -17,6 +17,20 @@
  */
 package org.fcrepo.persistence.ocfl.impl;
 
+import static org.apache.jena.graph.NodeFactory.createLiteral;
```

Comment:
Wonder if there's an option in intellij to not have it rearrange imports all the time?

Reply:
I adjusted my imports to try and match @dbernstein's setup, but I always have `static` imports at the top. I can manually revert these.